util.py

```diff
@@ -172,6 +172,8 @@ def download_zip_process(self, tablesToSync: dict) -> None:
                             zip_file.writestr(name, BytesIO(archive.read(name)).getvalue())
         except zipfile.BadZipfile as e:
             print(e)
+
+        #pathlib.Path('SOURCE.ZIP').write_bytes(zip_stream.getvalue())
```

Comment:
suggest to delete instead of commenting out code

Reply:
I will although its handy to have for debugging

Reply:
then maybe add comment , about how to use this for debugging